cherrypy/test/helper.py

```diff
@@ -226,23 +226,24 @@ def _setup_server(cls, supervisor, conf):
             # baseconf['server.ssl_module'] = 'builtin'
             baseconf['server.ssl_certificate'] = serverpem
             baseconf['server.ssl_private_key'] = serverpem
+        else:
+            baseconf['server.ssl_certificate'] = None
+            baseconf['server.ssl_private_key'] = None
 
         # helper must be imported lazily so the coverage tool
         # can run against module-level statements within cherrypy.
         # Also, we have to do "from cherrypy.test import helper",
         # exactly like each test module does, because a relative import
         # would stick a second instance of webtest in sys.modules,
         # and we wouldn't be able to globally override the port anymore.
-        if supervisor.scheme == 'https':
```

Comment:
why did you drop this?

Reply:
This would change HTTP_CON in the webtest.WebCase base class to HTTPS; all subsequent tests would use HTTPS and fail.  webtest.WebCase already configures its connection type properly on its own.